### PR TITLE
FINERACT-1568 Fix for Eclipse Link

### PR DESCRIPTION
**Pull request by josehernandezfintecheandomx**
## Description

In attention for the EclipseLink issue documented in the Ticket
[Apache Fineract JIRA ticket FINERACT-1568](https://issues.apache.org/jira/browse/FINERACT-1568).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java [hunk 1]
```diff
@@ -656,6 +665,8 @@ public CommandProcessingResult createDatatable(final JsonCommand command) {
             if (databaseTypeResolver.isMySQL()) {
                 sqlBuilder.append(" ENGINE=InnoDB DEFAULT CHARSET=UTF8MB4;");
             }
+            LOG.info("SQL:: {}", sqlBuilder.toString());
```
Comment (galovics) [inline fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java:668]: let's remove this log, or change it into debug

**Review by galovics — CHANGES_REQUESTED** [review]
A single minor change is needed but please make sure to rebase your changes with the latest develop.
The EclipseLink fix is already merged. Also, make sure to revert the build.gradle and persistence.xml changes.

**Comment by vidakovic** [comment]
@josehernandezfintecheandomx @galovics can we close this one or is there something happening here additionally?

**Comment by galovics** [comment]
> @josehernandezfintecheandomx @galovics can we close this one or is there something happening here additionally?

There's still a fix in this PR related to Datatable creation which we will need. Let's keep this open.

**Comment by galovics** [comment]
@josehernandezfintecheandomx also, do you mind adding a new integration testcase for validating the fixed functionality? Apparently it wasn't covered by the existing ones otherwise the build would've failed.

**Pull request closed without merging**